otcextensions/sdk/modelartsv1/v1/service.py

```diff
@@ -0,0 +1,299 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+import json
+
+from openstack import exceptions
+from openstack import resource
+
+
+class NodeSpec(resource.Resource):
+    #: Node architecture.
+    arch = resource.Body("arch")
+    #: Number of CPUs.
+    cpu = resource.Body("cpu", type=int)
+    #: Creation time, in the format of YYYY-MM-DDThh:mm:ss (UTC).
+    created_at = resource.Body("created_at")
+    #: Number of application instances deployed on a node.
+    deployment_num = resource.Body("deployment_num", type=int)
+    #: Description.
+    description = resource.Body("description")
+    #: Whether to enable the GPU.
+    enable_gpu = resource.Body("enable_gpu")
+    #: Number of GPUs.
+    gpu_num = resource.Body("gpu_num", type=int)
+    #: Host IP of a node.
+    host_ips = resource.Body("host_ips", type=list)
+    #: Host name of a node.
+    host_name = resource.Body("host_name")
+    #: Node ID.
+    id = resource.Body("id")
+    #: Status of a model instance on a node.
+    instance_status = resource.Body("instance_status")
+    #: Memory size, in MB.
+    memory = resource.Body("memory", type=int)
+    #: Node name.
+    name = resource.Body("name")
+    #: Node label.
+    node_label = resource.Body("node_label")
+    #: OS name.
+    os_name = resource.Body("os_name")
+    #: OS type.
+    os_type = resource.Body("os_type")
+    #: OS version.
+    os_version = resource.Body("os_version")
+    #: Inference URL of a node.
+    predict_url = resource.Body("predict_url")
+    #: Host status.
+    state = resource.Body("state")
+    #: Update time, in the format of YYYY-MM-DDThh:mm:ss (UTC).
+    updated_at = resource.Body("updated_at")
+
+
+class CustomSpec(resource.Resource):
+    #: Number of required CPUs.
+    cpu = resource.Body("cpu", type=float)
+    #: Number of GPUs, which can be decimals.
+    gpu_p4 = resource.Body("gpu_p4", type=float)
+    #: Required memory capacity, in MB.
+    memory = resource.Body("memory", type=int)
+
+
+class ScheduleSpec(resource.Resource):
+    #: Value that maps to the time unit.
+    duration = resource.Body("duration", type=int)
+    #: Scheduling time unit.
+    time_unit = resource.Body("time_unit")
+    #: Scheduling type.
+    type = resource.Body("type")
+
+
+class ConfigSpec(resource.Resource):
+    #: Additional model deployment attribute.
+    additional_properties = resource.Body("additional_properties", type=dict)
+    #: ID of a dedicated resource pool.
+    cluster_id = resource.Body("cluster_id")
+    #: Custom specifications.
+    custom_spec = resource.Body("custom_spec", type=CustomSpec)
+    #: OBS path of the output data of a batch job.
+    dest_path = resource.Body("dest_path")

```

Comment:
destination?

Reply:
dest_path name should be fine for destination_path